src/System.CodeDom/ref/System.CodeDom.cs

```diff
@@ -994,7 +994,7 @@ public abstract partial class CodeGenerator : System.CodeDom.Compiler.ICodeGener
         protected abstract void GenerateSnippetMember(System.CodeDom.CodeSnippetTypeMember e);
         protected virtual void GenerateSnippetStatement(System.CodeDom.CodeSnippetStatement e) { }
         protected void GenerateStatement(System.CodeDom.CodeStatement e) { }
-        protected void GenerateStatements(System.CodeDom.CodeStatementCollection stms) { }
+        protected void GenerateStatements(System.CodeDom.CodeStatementCollection stmts) { }
```

Comment:
Note, this is a discrepancy between source and ref (and is a breaking change). Should this rename be reverted?

Reply:
Per https://referencesource.microsoft.com/#System/compmod/system/codedom/compiler/CodeGenerator.cs,1adc9125a4b2c602 `stms` is correct (aka the NetFx answer) and the /src/ here should be corrected.